networkx/algorithms/bipartite/tests/test_spectral_bipartivity.py

```diff
@@ -79,13 +79,13 @@ def k23_like(self):
     def test_single_nodes(self):
 
         # single nodes
-        G=nx.complete_bipartite_graph(2,3)
+        G=nx.complete_bipartite_graph(2, 3)
         G.add_edge(2,4)
         sbn=sb(G,nodes=[1,2])
         assert_almost_equal(sbn[1],0.85,places=2)
         assert_almost_equal(sbn[2],0.77,places=2)
 
-        G=nx.complete_bipartite_graph(2,3)
+        G=nx.bipartite.complete_bipartite_graph(2,3)
```

Comment:
`complete_bipartite_graph` is imported to the main namespace, so the this should be `G=nx.complete_bipartite_graph(2,3)`.